sink/datadog.go

```diff
@@ -276,9 +289,30 @@ func (s *Datadog) Send(ctx context.Context, m *blip.Metrics) error {
 
 			// Convert Blip metric type to Datadog metric type
 			switch metrics[i].Type {
-			case blip.COUNTER:
+			case blip.CUMULATIVE_COUNTER, blip.DELTA_COUNTER:
```

Comment:
Seems like you need only `case blip. CUMULATIVE_COUNTER`? If it's already a delta, then no extra work is needed? (Or, if delta, then just `metricVal := metrics[i].Value`.)

Reply:
Would we really need to put a delta value in the map? It seems like if it is `DELTA_COUNTER` that we want to avoid adding it to the map to reduce memory overhead.

Reply:
I am not putting the `DELTA_COUNTER` in the map, I have only merged them into a single case as they have 15+ line in common which I will otherwise have to duplicate. But if it's confusing, I can make them seperate cases.

The extra calculation required for `CUMULATIVE_COUNTER`s is branched with 

```
if s.sendCounterDelta && metrics[i].Type == blip.CUMULATIVE_COUNTER {
```

The value is sent as is, if its a `DELTA_COUNTER`